### PR TITLE
K1 7 view contract performance Merge

### PR DESCRIPTION
**Pull request by LazuliSound**

### CODE REVIEW

File: app/src/main/java/id/ac/itb/ditlog/monitorandperformance/AsyncAddIndicator.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
  */
 
 public class AsyncAddIndicator extends AsyncTask<String,Void,String> {
+    public String auth = "eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJhZXAiLCJyb2xlSWQiOjQyMiwiZXhwIjoxNTIyMzM2Mzg1fQ.3nai_tQNWLObap18t8YjZ-RrtisOhlPLq7kI_zDgy1Gq99VNdWTicQ5o-c8BPTh2ZPRxBOhIqumAaCc-8F9-2A";
     public static final String SERVER_URL = "159.65.131.168:8080";
```
Comment (keychera) [inline app/src/main/java/id/ac/itb/ditlog/monitorandperformance/AsyncAddIndicator.java]: erase this, use `BuildConfig.WEBSERVICE_URL` instead to refer the webservice url

File: app/src/main/java/id/ac/itb/ditlog/monitorandperformance/AsyncAddIndicator.java [hunk 2]
```diff
@@ -45,6 +46,7 @@ protected String doInBackground(String... params) {
 
                 URL url = new URL("http://" + SERVER_URL + "/indicators");
```
Comment (keychera) [inline app/src/main/java/id/ac/itb/ditlog/monitorandperformance/AsyncAddIndicator.java]: use `BuildConfig.WEBSERVICE_URL` instead to refer the webservice url

File: app/src/main/java/id/ac/itb/ditlog/monitorandperformance/ChooseIndicator.java [hunk 3]
```diff
@@ -211,13 +211,14 @@ public void onCheckboxClicked(View view) {
 
     public class indicatorGetter extends AsyncTask<Void, Void, ArrayList<IndicatorEntity>>{
 
+        public String auth = "eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJhZXAiLCJyb2xlSWQiOjQyMiwiZXhwIjoxNTIyMzM2Mzg1fQ.3nai_tQNWLObap18t8YjZ-RrtisOhlPLq7kI_zDgy1Gq99VNdWTicQ5o-c8BPTh2ZPRxBOhIqumAaCc-8F9-2A";
         public static final String SERVER_URL = "159.65.131.168:8080";
```
Comment (keychera) [inline app/src/main/java/id/ac/itb/ditlog/monitorandperformance/ChooseIndicator.java]: erase this, use `BuildConfig.WEBSERVICE_URL` instead to refer the webservice url

File: app/src/main/java/id/ac/itb/ditlog/monitorandperformance/ChooseIndicator.java [hunk 4]
```diff
@@ -238,6 +239,7 @@ public void onCheckboxClicked(View view) {
                 URL url = new URL(rawUrl);
```
Comment (keychera) [inline app/src/main/java/id/ac/itb/ditlog/monitorandperformance/ChooseIndicator.java:239]: change `SERVER_URL` to  `BuildConfig.WEBSERVICE_URL` to refer the webservice url

File: app/src/main/java/id/ac/itb/ditlog/monitorandperformance/ViewContractPerformance.java [hunk 5]
```diff
@@ -0,0 +1,410 @@
...
+                    public void onNothingSelected(AdapterView<?> parent) {
+                        //showToast("Spinner1: unselected");
+                    }
+                });
+        return true;
+    }
+
+    public class AsyncGetContracts extends AsyncTask<Void, Void, ArrayList<ContractEntity>>{
+        public String auth = "eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJhZXAiLCJyb2xlSWQiOjQyMiwiZXhwIjoxNTIyMzM2Mzg1fQ.3nai_tQNWLObap18t8YjZ-RrtisOhlPLq7kI_zDgy1Gq99VNdWTicQ5o-c8BPTh2ZPRxBOhIqumAaCc-8F9-2A";
+        private static final String SERVER_URL = "159.65.131.168:8080";
```
Comment (keychera) [inline app/src/main/java/id/ac/itb/ditlog/monitorandperformance/ViewContractPerformance.java]: erase this, use `BuildConfig.WEBSERVICE_URL` to refer the webservice url

File: app/src/main/java/id/ac/itb/ditlog/monitorandperformance/ViewContractPerformance.java [hunk 6]
```diff
@@ -0,0 +1,410 @@
...
+            this.year = year;
+            this.context = context;
+            this.act = act;
+        }
+
+        @Override
+        protected ArrayList<ContractEntity> doInBackground(Void... voids) {
+            String method = "GET";
+            try {
+                String rawUrl = "http://" + SERVER_URL + "/contracts?tahun=" + year;
```
Comment (keychera) [inline app/src/main/java/id/ac/itb/ditlog/monitorandperformance/ViewContractPerformance.java]: change `SERVER_URL` to `BuildConfig.WEBSERVICE_URL` to refer the webservice url

**Review by keychera — CHANGES_REQUESTED** [review]
need modification
- erase static string `SERVER_URL`, and use `BuildConfig.WEBSERVICE_URL` instead

**Review by dayuputuari — APPROVED** [review]
View contract performance has been working. Need integration with login and authorization token.

**Review by keychera — APPROVED** [review]

**Pull request merged**